release/internal/hashrelease/pinnedversion.go

```diff
@@ -48,14 +49,14 @@ func (c Component) String() string {
 
 // PinnedVersionData represents the data needed to generate the pinned version file.
 type PinnedVersionData struct {
-	ReleaseName   string
-	BaseDomain    string
-	CalicoVersion string
-	Operator      Component
-	Note          string
-	Hash          string
-	Registry      string
-	ReleaseBranch string
+	ReleaseName    string
+	BaseDomain     string
+	ProductVersion string
```

Comment:
Personally I find this change from calico -> product to be more confusing than helpful. This is meant to be the calico repository version, right?

Reply:
This change is to avoid conflicts with close-source since that will have the "CalicoVersion" mean something else